config/jobs/cert-manager/cert-manager/release-previous/cert-manager-release-previous-presubmits.yaml

```diff
@@ -69,99 +70,142 @@ presubmits:
+        - mountPath: /home/prow/go/pkg/mod
+          name: gopkgmod
+        - mountPath: /home/prow/go/src/github.com/cert-manager/cert-manager/bin/downloaded
+          name: bindownloaded
+      volumes:
+      - name: gocache
+        hostPath:
+          path: /tmp/gocache
+          type: DirectoryOrCreate
+      - name: gopkgmod
+        hostPath:
+          path: /tmp/gopkgmod
+          type: DirectoryOrCreate
+      - name: bindownloaded
+        hostPath:
+          path: /tmp/bindownloaded
+          type: DirectoryOrCreate
       dnsConfig:
         options:
           - name: ndots
             value: "1"
 
-  - name: pull-cert-manager-deps
-    always_run: true
-    context: pull-cert-manager-deps
+  - name: pull-cert-manager-e2e-v1-18
+    context: pull-cert-manager-e2e-v1-18
+    always_run: false
+    optional: true
     max_concurrency: 4
     agent: kubernetes
     decorate: true
     branches:
+    # cert-manager 1.8 supports k8s 1.19+, so no need to run against release-1.8 here
     - release-1.7
-    - release-1.6
     annotations:
       testgrid-create-test-group: 'false'
-      description: Verifies dependency related files are up to date
+      description: Runs the end-to-end test suite against a Kubernetes v1.18 cluster
     labels:
       preset-service-account: "true"
+      preset-dind-enabled: "true"
       preset-bazel-remote-cache-enabled: "true"
       preset-bazel-scratch-dir: "true"
+      preset-cloudflare-credentials: "true"
+      preset-retry-flakey-tests: "true"
+      preset-disable-all-alpha-feature-gates: "true"
+      preset-ginkgo-skip-default: "true"
     spec:
       containers:
       - image: eu.gcr.io/jetstack-build-infra-images/bazelbuild:20210831-4cf7b0b-4.2.1
         args:
         - runner
-        - make
-        - verify_deps
+        - devel/ci-run-e2e.sh
         resources:
           requests:
-            cpu: 2
-            memory: 4Gi
+            cpu: 3500m
+            memory: 12Gi
+        env:
+        - name: K8S_VERSION
+          value: "1.18"
+        securityContext:
+          privileged: true
+          capabilities:
+            add: ["SYS_ADMIN"]
+        volumeMounts:
+        - mountPath: /lib/modules
+          name: modules
+          readOnly: true
+        - mountPath: /sys/fs/cgroup
+          name: cgroup
+      volumes:
+      - name: modules
+        hostPath:
+          path: /lib/modules
+          type: Directory
+      - name: cgroup
+        hostPath:
+          path: /sys/fs/cgroup
+          type: Directory
       dnsConfig:
         options:
-          - name: ndots
-            value: "1"
+        - name: ndots
+          value: "1"
 
-  # cert-manager 1.7 requires at least K8S 1.18 so this will be run only for the release-1.6
-  # branch.
-  - name: pull-cert-manager-e2e-v1-17
-    context: pull-cert-manager-e2e-v1-17
-    always_run: true
+  - name: pull-cert-manager-e2e-v1-19
+    context: pull-cert-manager-e2e-v1-19
+    always_run: false
```

Comment:
For this, my thinking was that we'd need to have two versions of each test, one for release-1.7 with Bazel, one with release-1.8 with make and since the presubmits will only be run a few times max to get some bug fixes merged, it shouldn't really matter whether it uses make or Bazel.

However, perhaps it would cause problems down the line that the periodics use make whilst the presubmits use Bazel.. I've had an issue with those being different for periodics and presubmits against master, where a PR for a flake for make periodic could not be tested by a presubmit because there was only a Bazel presubmit for that version of Kubernetes.

So I will add make version for 1.8 in this PR